ui/app/routes/datasets/$dataset_name/datapoint/$id/DatapointActions.tsx

```diff
@@ -30,18 +30,18 @@ export function DatapointActions({
   canSave,
   isEditing,
   onReset,
-  showTryWithVariant,
+  showTryWithButton,
 }: DatapointActionsProps) {
   const handleCancel = () => {
     onReset();
     toggleEditing();
   };
   return (
     <ActionBar>
-      {showTryWithVariant && (
-        <TryWithVariantButton
-          variants={variants}
-          onVariantSelect={onVariantSelect}
+      {showTryWithButton && (
+        <TryWithButton
+          options={variants}
+          onOptionSelect={onVariantSelect}
           isLoading={variantInferenceIsLoading}
```

Comment:
Since `showTryWithButton` only gets set to true if the `function_name` is not `"tensorzero::default"`, I don't think we need to do anything here.

Reply:
Created a new issue: https://github.com/tensorzero/tensorzero/issues/3569